modules/gcloud/src/test/scala/com/dimafeng/testcontainers/PubSubEmulatorContainerSpec.scala

```diff
@@ -0,0 +1,105 @@
+package com.dimafeng.testcontainers
+
+import com.google.cloud.pubsub.v1.{AckReplyConsumer, MessageReceiver}
+import com.google.protobuf.ByteString
+import com.google.pubsub.v1._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+import java.util.concurrent.TimeUnit
+import scala.collection.JavaConverters._
+import scala.concurrent.TimeoutException
+
+class PubSubEmulatorContainerSpec extends AnyWordSpecLike with Matchers with ForAllTestContainer {
+
+  override val container: PubSubEmulatorContainer = PubSubEmulatorContainer()
```

Comment:
I see that `defaultImageName` uses the latest tag which is fine for the general use. But can we pin the version in the test (so it's not gets broken unpredictably)?

Reply:
Sounds reasonable yes. I locked version `382.0.0` in latest commit so image is unaltered.